packages/@eventual/core-runtime/src/handlers/command-worker.ts

```diff
@@ -96,41 +97,44 @@ function initRouter() {
       // RPC route takes a POST request and passes the parsed JSON body as input to the input
       router.post(
         commandRpcPath(command),
-        withMiddleware(async (request, context) => {
-          if (command.passThrough) {
-            // if passthrough is enabled, just proxy the request-response to the handler
-            return command.handler(request, context);
-          }
+        withMiddlewares<CommandContext, HttpResponse, HttpRequest>(
+          command.middlewares ?? [],
```

Comment:
What changed here?

Reply:
Generalized the function to work for both socket and command.